src/app/public/modules/flyout/flyout.component.html

```diff
@@ -64,18 +107,30 @@
   <ng-template [ngIf]="permalink">
     <ng-template [ngIf]="permalink.url">
       <a
-        class="sky-btn sky-btn-default sky-flyout-btn-permalink sky-margin-inline-default"
+        class="sky-btn sky-flyout-btn-permalink sky-margin-inline-default"
         [href]="permalink.url"
+        [ngClass]="{
+          'sky-btn-default':
+            (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'default',
+          'sky-btn-borderless':
+            (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'modern'
+        }"
         (click)="close()"
       >
         {{ permalinkLabel }}
       </a>
     </ng-template>
     <ng-template [ngIf]="permalink.route">
       <a
-        class="sky-btn sky-btn-default sky-flyout-btn-permalink sky-margin-inline-default"
+        class="sky-btn sky-flyout-btn-permalink sky-margin-inline-default"
         [routerLink]="permalink.route.commands"
         [fragment]="permalink.route.extras?.fragment"
+        [ngClass]="{
+          'sky-btn-default':
+            (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'default',
+          'sky-btn-borderless':
+            (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'modern'
```

Comment:
There are nine references to `themeSvc.settingsChange` subscription. What do you think about doing the same thing for this component as you did above?

Reply:
Done.